searx/searx
# [fix] simple theme: fix autocomplete

## What does this PR do?

AJAX requests send the X-Requested-With HTTP header,
so searx.webapp.autocompleter returns the results with the expected data format.

## Why is this change important?

Since PR #2127, with the simple theme, autocomplete doesn't work 

## How to test this PR locally?

1. Use the simple
1. Try autocomplete in the form
1. Check it is working
1. Add searx as the default engine
1. Check autocomplete is working (in the browser search bar)

## Author's checklist



## Related issues

Related to #2127
Close #2203
